app/models/mixins/ansible_authentications.rb

```diff
@@ -25,6 +25,7 @@ def ansible_secret_key
   def ansible_secret_key=(key)
     auth = authentication_for_type(ANSIBLE_SECRET_KEY_TYPE, "Ansible Secret Key")
     update_ansible_authentication(auth, :auth_key => key)
+    key
```

Comment:
This is not necessary as any setter method (ending in `=`) automatically returns the incoming param

```
$ irb
[1] pry(main)> def blah=(x)
[1] pry(main)*   "HI"
[1] pry(main)* end
=> :blah=
[2] pry(main)> blah = "NOT HI"
=> "NOT HI"
```